lib/utils.js

```diff
@@ -77,17 +77,25 @@ async function getAndCheckIosSdkVersion () {
 
 async function translateDeviceName (pv, dn = '') {
   let deviceName = dn;
-  if (dn.toLowerCase() === 'iphone simulator') {
+  dn = dn.toLowerCase().trim();
+  if (dn === 'iphone simulator') {
     deviceName = 'iPhone 6';
-  } else if (dn.toLowerCase() === 'ipad simulator') {
+  } else if (dn === 'ipad simulator') {
     if (parseFloat(pv).toFixed(1) < "10.3") {
```

Comment:
I didn't know that strings can be compared this way o_O

Reply:
Well... it compares lexically. so it won't work for `pv = 9.3`, since 9 is greater than 1.

Reply:
This should not be done in this way, is the takeaway. :)

Reply:
I'll fix.

Reply:
yeah, if we're converting to float anyway might as well do the comparison on a number